asv_bench/benchmarks/array.py

```diff
@@ -1,7 +1,10 @@
 import numpy as np
+import pyarrow as pa
```

Comment:
not sure if the policy should change, but AFAIK we guard the pyarrow import in the other benchmarks as it's an optional dependency and raise NotImplementedError so that the benchmarks get skipped when pyarrow not installed.